documentation/modules/upgrading/proc-upgrade-cli-tool-direct.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// assembly-upgrade-resources.adoc
+
+[id='proc-upgrade-cli-tool-direct-{context}']
+= Converting custom resources directly using the API conversion tool
+
+[role="_abstract"]
+This procedure describes how to use the API conversion tool to convert Strimzi custom resources directly in the kubernetes cluster into a format applicable to `{ApiVersion}`.
+To do so, you use the `convert-resource` (`cr`) command.
+The command uses Kubernetes APIs to make the conversions.
+
+You can specify one or more of types of Strimzi custom resources, based on the `kind` property, or you can convert all types.
+You can also target a specific namespace or all namespaces for conversion.
+When targeting a namespace, you can convert all custom resources in that namespace, or convert a single custom resource by specifying its name and kind.
+
```

Comment:
Should we add link to the `convert-file` procedure as an alternative?

Reply:
I've added links to and from each